src/PIL/EpsImagePlugin.py

```diff
@@ -312,14 +312,14 @@ def _find_offset(self, fp):
             fp.seek(0, io.SEEK_END)
             length = fp.tell()
             offset = 0
-        elif i32(s[0:4]) == 0xC6D3D0C5:
+        elif i32(s, 0) == 0xC6D3D0C5:
```

Comment:
```suggestion
        elif i32(s) == 0xC6D3D0C5:
```

Reply:
I’m in favor of being explicit for clarity of what is being converted.